WTWSMS/localisation/AO_events.csv

```diff
@@ -187,7 +187,7 @@ ruthless_desc;This character believes that the ends justify the means, and will
 honorable;Honorable;Honorable;;;;;;;;;;;;x
 honorable_desc;This character is a firm believer in the importance of duty, mercy, and honor.;Ce personnage croit fermement en l'importance du devoir, de la piti� et de l'honneur.;;;;;;;;;;;;x
 plot_kidnap_character_title;Kidnap [This.GetTitledFirstName];Kidnapper [This.GetTitledFirstName];[This.GetTitledFirstName] entf�hren;;;;;;;;;;;x
-plot_kidnap_character_short;Kidnap [This.GetTitledFirstName];Kidnapper [This.GetTitledFirstName];;;;;;;;;;;;x
+plot_kidnap_character_short;Plot to Kidnap;Kidnapper [This.GetTitledFirstName];;;;;;;;;;;;x
```

Comment:
Why did you change this?

Reply:
It's broken, returns your character instead of target

Reply:
Should be FROM as the scope, no?

Reply:
I think I tested with FROM and it didn't work.

Reply:
What happens with [From.GetTitledFirstName]?

Reply:
That is what I used, From not FROM